fr.masson_informatique.WhosWho.yml

```diff
@@ -0,0 +1,241 @@
+app-id: fr.masson_informatique.WhosWho
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: whoswho
+modules:
+  - name: WhosWho
+    buildsystem: simple
+    build-commands:
+      - ./build-translations.py
+      - pip3 install --prefix=${FLATPAK_DEST} --no-deps .
```

Comment:
why `--no-deps` ?

Reply:
If I understood correctly, it prevents pip from automatically downloading dependencies, which does not work inside the "flatpak-builder sandbox". Am I wrong?

Reply:
You are right, this seems logical, I have put whoswho's module at the end.